controlplane/kubeadm/internal/etcd/util/set.go

```diff
@@ -0,0 +1,214 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Modified copy of k8s.io/apimachinery/pkg/util/sets/int64.go
+// Modifications
+//   - int64 became uint64
+//   - Empty type is added to this file and changed to a non exported symbol
+
+package util
+
+import (
+	"reflect"
+	"sort"
+)
+
+type empty struct{}
+
+// util.UInt64 is a set of int64s, implemented via map[uint64]struct{} for minimal memory consumption.
+type UInt64 map[uint64]empty
+
+// NewUInt64 creates a UInt64 from a list of values.
+func NewUInt64(items ...uint64) UInt64 {
+	ss := UInt64{}
+	ss.Insert(items...)
+	return ss
+}
+
+// Int64KeySet creates a UInt64 from a keys of a map[uint64](? extends interface{}).
+// If the value passed in is not actually a map, this will panic.
+func Int64KeySet(theMap interface{}) UInt64 {
```

Comment:
Is this used or going to be used anywhere? If not, I'd remove it given it can panic

Reply:
unused, happy to remove

Reply:
Thanks